### ISSUE TITLE
`wee_import` should allow import of text fields

### ISSUE THREAD

**Issue by gjr80** [weewx/weewx#732]
As WeeWX accepts text fields (fields of type TEXT in the database archive table schema) it stands that `wee_import` should support importing text fields.

[This weewx-user thread](https://groups.google.com/g/weewx-user/c/afPbV04SDjs) refers.

**Comment by gjr80** [comment]
`wee_import` presently supports imports from WeatherUnderground, WeatherDisplay, Cumulus, WeatherCat and CSV format files. All sources except CSV format files use fixed format data sources/files that do not include text fields. CSV imports are not constrained in this manner and could contain text fields. Consequently, only CSV imports need support text field imports at this time. If the other sources include text fields it should be a simple matter to extend support of CSV text field import to other sources.

CSV imports presently require each CSV field that is imported to be assigned a WeeWX unit type via the import config file field map. There is no unit type that supports text fields. One approach would be to use the literal string 'text' as the 'unit type' for text fields in the field map. So a user would define a field map to import text field `Comment` as WeeWX archive field `comment` as follows:
```
    [[FieldMap]]
        ....
        comment = Comment, text
```

**Comment by gjr80** [comment]
Created the `wee_import_text` branch off `development` and have implemented support for import of CSV text fields.

**Comment by gjr80** [comment]
First cut implementation suffered from the error outlined in [this weewx-user post](https://groups.google.com/g/weewx-user/c/afPbV04SDjs/m/qXgeIsZ-BQAJ).

This is due to to `wee_import` not being cognisant of the accumulator config in the `[Accumulators]` stanza in `weewx.conf`. Fixed at omnibus commit 7a0518c2f0f1e09ebe24d90b4efc2d770283e0c3.

**Comment by gjr80** [comment]
Correct operation of updated `wee_import` code confirmed in [this weewx-user post](https://groups.google.com/g/weewx-user/c/afPbV04SDjs/m/1PWqOFmgBQAJ).

Documentation and changelog updated. To appear in v4.7.1
